### PR TITLE
Clarification on internal_url

### PR DESCRIPTION
**Pull request by tubalainen**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/home_connect.markdown [hunk 1]
```diff
@@ -34,7 +34,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 - Application ID: Home Assistant (or whatever name makes sense to you)
 - OAuth Flow: Authorization Code Grant Flow
-- Redirect URI  : `<YOUR_HOME_ASSISTANT_URL>/auth/external/callback`
+- Redirect URI  : `<YOUR_INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use same the address defined as `internal_url` in your configuration settings. Example `http://192.168.1.24:8123/auth/external/callback`
```
Comment (frenck) [inline source/_integrations/home_connect.markdown]: I think this doesn't make this clearer. As one does not have to define an internal URL in the settings. So what if I didn't set it?

Comment (tubalainen) [reply source/_integrations/home_connect.markdown]: Ok, got your point on the config part.

Then just mention that it needs to be the local IP-address URL? I personally fell into that "trap" today trying to use my nabucasa URL.

Comment (tubalainen) [reply source/_integrations/home_connect.markdown]: More confusion on the subject can be found here: https://community.home-assistant.io/t/0-110-home-connect-not-working/197602/13

Comment (frenck) [reply source/_integrations/home_connect.markdown]: It doesn't have to be an IP-Address... `homeassistant.local` is a perfectly fine host to use in an internal URL. Or in my case, it is even a subdomain of my personal main domain that resolves locally.

**Review by tubalainen — COMMENTED** [review]
Would this work? @frenck

File: source/_integrations/home_connect.markdown (hunk 1 above)
Comment (tubalainen) [reply source/_integrations/home_connect.markdown]: Ok, updated. Please check :)

File: source/_integrations/home_connect.markdown [hunk 2]
```diff
@@ -34,7 +34,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 - Application ID: Home Assistant (or whatever name makes sense to you)
 - OAuth Flow: Authorization Code Grant Flow
-- Redirect URI  : `<YOUR_INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use same the address defined as `internal_url` in your configuration settings. Example `http://192.168.1.24:8123/auth/external/callback`
+- Redirect URI  : `<YOUR_LOCAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use your local IP address or hostname. Example `http://homeassistant.local:8123/auth/external/callback`
```
Comment (frenck) [inline source/_integrations/home_connect.markdown]: How about this?

```suggestion
- Redirect URI: "`<INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback`
  Use your internal Home Assistant URL, if you didn't configure one manually, use your local IP address. Examples: `http://192.168.0.2:8123/auth/external/callback`, `http://homeassistant.local:8123/auth/external/callback`."
```

Comment (tubalainen) [reply source/_integrations/home_connect.markdown]: Sure. That should clarify most of the configuration scenarios. :) 
The important thing is for our users to not struggle with the component trying to get it going.

**Review by frenck — APPROVED** [review]
Thanks for the help here @tubalainen!

👍

**Pull request merged**